test/query_builder_test.dart

```diff
@@ -0,0 +1,38 @@
+import 'dart:math';
+
+import 'package:angular_test/angular_test.dart';
+import 'package:test/test.dart';
+
+import 'package:dv360_excel_plugin/src/query_builder.dart';
+
+void main() {
+  const max = 10000;
+
+  Random random;
+  QueryBuilder queryBuilder;
+
+  setUp(() {
+    random = Random();
```

Comment:
i see what you are going for but we typically stay away from randomness in tests because it can cause flakiness issues. Lets think about another way to test the intent here.